tools/flang2/flang2exe/iliutil.cpp

```diff
@@ -2102,7 +2102,7 @@ reciprocal_division(int n, INT divisor, int sgnd)
   } else {
     INT64 twol;
     int shpre = 0;
-    INT64 one_64 = {0x0, 0x1}, divisor64 = {0x0, udiv};
+    INT64 one_64 = {0x0, 0x1}, divisor64 = {0x0, divisor};
```

Comment:
That's fairly subtle!  You might want to merge the three if(sgnd) statements into one for clarity.

Reply:
ok